frontend/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+    "parameters": {
+        "domainAdminUsername": {
+            "value": "GEN-USER"
+        },
+        "domainAdminPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "domainName": {
+            "value": "mydomain.com"
+        },
+        "registrationCode": {
+            "value": "xxxxxxxxxxxx@nnnn-nnnn-nnnn-nnnn"
+        },
+        "AzureAdminUsername": {
+            "value": "username@example.com"
+        },
+        "AzureAdminPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "tenantID": {
+            "value": "null"
+        },
+        "CAMDeploymentBlobSource": {
+            "value": "https://teradeploy.blob.core.windows.net/binaries"
+        },
+        "_agentArtifactsLocation": {
+            "value": "https://raw.githubusercontent.com/teradici/deploy/TSW-66292-split-ARM-template-cleanup/end-user-application-machines/new-agent-vm"
```

Comment:
Same as the other parameters files - I think we should make them all work after being merged to master.